suse_migration_services/units/regenerate_initrd.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2022 SUSE Linux LLC.  All rights reserved.
+#
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>
+#
+import logging
+
+# project
+from suse_migration_services.command import Command
+from suse_migration_services.defaults import Defaults
+from suse_migration_services.logger import Logger
+from suse_migration_services.migration_config import MigrationConfig
+
+from suse_migration_services.exceptions import (
+    DistMigrationCommandException
+)
+
+
+def main():
+    """
+    DistMigration create a new initrd with added modules
+
+    Run dracut to build a new initrd that includes multipath modules
+    """
+    Logger.setup()
+    log = logging.getLogger(Defaults.get_migration_log_name())
+
+    if MigrationConfig().is_host_independent_initd_requested():
+        log.info('Creating a new host independent initrd')
+        root_path = Defaults.get_system_root_path()
+
+        do_bind_mounts(root_path)
+        run_dracut(root_path)
+
+
+def do_bind_mounts(root_path,):
```

Comment:
bit nickpicking,
maybe rename to `dracut_bind_mounts` or something more specific for this case ?
as there are other bind mounts in the codebase

Another option would be refactor into having a common bind mounting method but not the purpose of this PR
and probably no much benefit

Reply:
sure I will change this

Reply:
fixed, @jesusbv please re-review